ci/scripts/run_smoke_test.js

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env node
+
+const AWS = require('aws-sdk')
+const synthetics = new AWS.Synthetics()
+const s3 = new AWS.S3()
+const CHECK_INTERVAL = 5000
+const { SMOKE_TEST_NAME } = process.env
+
+async function describeCanariesLastRun () {
+  return synthetics.describeCanariesLastRun({}).promise()
+}
+
+function runCanary () {
+  console.log(`Starting canary: ${SMOKE_TEST_NAME}`)
+  // 'startCanary' should run it once then stop, according to its schedule config in terraform
+  return synthetics.startCanary({ Name: SMOKE_TEST_NAME }).promise()
+}
+
+function prettyPrintRunReport (runReport) {
+  console.log(`Name: ${runReport.Name}`)
+  console.log(`Status: ${runReport.Status.State}`)
+  if (runReport.Status.State === 'FAILED') {
+    console.log(`Failure Reason: ${runReport.Status.StateReason}`)
+  }
+}
+
+async function getS3Objects (splitLocation, prefix) {
+  return s3.listObjects({ Bucket: splitLocation[0], Prefix: prefix }).promise()
+}
+
+async function run () {
+  const startedAt = Date.now()
+  try {
+    await runCanary()
+  } catch (error) {
+    console.error(error)
+    process.exitCode = 1
```

Comment:
Not sure which is best practice (interested to know), but I've used the following before rather than `process.exitCode = 1` and `return`
```
process.exit(1)
```